packages/gboost-infra/src/file-upload/function/getUploadURL.ts

```diff
@@ -24,15 +24,29 @@ export async function getUploadURL(params: getUploadURLParams) {
   const { logger } = params;
 
   if (process.env.BUCKET_MAP) {
-    const command = new PutObjectCommand({
-      Bucket: JSON.parse(process.env.BUCKET_MAP)[bucket].bucket,
-      Key: JSON.parse(process.env.BUCKET_MAP)[bucket].key + fileName,
-    });
-    const url = await getSignedUrl(client, command, { expiresIn: 3600 });
+    const bucketMap = JSON.parse(process.env.BUCKET_MAP);
+    let i = 0;
+    let notFound = true;
+    while (notFound && i < bucketMap.length) {
+      if (bucketMap[i].bucket === bucket) {
+        notFound = false;
+      } else {
+        i++;
+      }
+    }
```

Comment:
Could you export this functionality into a utility function instead of duplicating the code in getUploadPartURL, getUploadId, ...?